### PR TITLE
Change structure adjust style

### PR DESCRIPTION
**Pull request by cmaiw**

### CODE REVIEW

File: client/src/components/NavLinkStyled.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
 import styled from '@emotion/styled';
-import React from 'react';
+import { NavLink } from 'react-router-dom';
```
Comment (leonp5) [inline client/src/components/NavLinkStyled.js]: I'm not sure if you could import `NavLink` from react-router-dom
Maybe you mean `Link`?

File: client/src/components/NavLinkStyled.js [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
 
-const ThisNavButton = styled.button`
-  display: flex;
-  height: 40px;
-  width: 40px;
-  justify-content: center;
+const NavLinkStyled = styled(NavLink)`
```
Comment (leonp5) [inline client/src/components/NavLinkStyled.js]: Then you could write here `NavLinkStyled = styled(Link)`

File: db.json [hunk 3]
```diff
@@ -1,15 +0,0 @@
-{
```
Comment (leonp5) [inline db.json]: You put db.json on the .gitignore
Nice :)

File: client/src/components/NavLinkStyled.js (hunk 1 above)
Comment (cmaiw) [reply client/src/components/NavLinkStyled.js]: Thank you. But you can import this component. Checkout: https://reacttraining.com/react-router/web/api/NavLink

File: db.json (hunk 3 above)
Comment (cmaiw) [reply db.json]: Thanks ;) ...

**Pull request merged**